### PR TITLE
removes django-compressor and edits scss templates for webpack #10533

### PR DESCRIPTION
**Pull request by chrabyrd**
# DO NOT MERGE! THIS HAS A TEMPORARY ARCHES-DEV-DEPENDENCY. AFTER BRANCH APPROVAL `package.json` NEEDS TO BE REVERTED AND THIS BRANCH AND https://github.com/archesproject/arches-dev-dependencies/pull/12 NEED TO BE MERGED CONCURRENTLY

### Types of changes

-   [ ] Bugfix (non-breaking change which fixes an issue)
-   [x] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Issues Solved

#10533

### CODE REVIEW

**Review by apeters — APPROVED** [review]
looks good to me, you'll want to update package.json to point to dev/7.6.x before we merge into dev/7.6.x

File: arches/app/media/css/arches.scss [hunk 1]
```diff
@@ -23,12 +23,14 @@
...
 @import url(../css/base-manager.css);
-@import "css/abstracts";
-@import "sidenav.scss";
-@import "edit-history.scss";
-@import "tree.scss";
-@import "jqtree.scss";
-@import "rdm.scss";
+@import "abstracts/breakpoint-settings";
+@import "abstracts/functions";
+@import "abstracts/index";
```
[the remark this answers is not in the document]
Comment (chrabyrd) [reply arches/app/media/css/arches.scss:28]: ah -- I had to un-minify this file to fix some minor issues with some written CSS  in it being deprecated and breaking the compiler

File: arches/app/media/css/arches.scss [hunk 2]
```diff
@@ -1,8 +1,8 @@
-@import url(fonts/openSans.css);
+@import url(../fonts/openSans.css);
 @import url(../node_modules/bootstrap/dist/css/bootstrap.min.css);
 @import url(../node_modules/eonasdan-bootstrap-datetimepicker/build/css/bootstrap-datetimepicker.min.css);
 @import url(../node_modules/bootstrap-colorpicker/dist/css/bootstrap-colorpicker.min.css);
-@import url(nifty.min.css);
```
[the remark this answers is not in the document]
Comment (chrabyrd) [reply arches/app/media/css/arches.scss:5]: ah -- I had to un-minify this file to fix some minor issues with some written CSS  in it being deprecated and breaking the compiler

File: arches/app/media/css/arches.scss (hunk 1 above)
[the remark this answers is not in the document]
Comment (chrabyrd) [reply arches/app/media/css/arches.scss]: fixed in latest 👍

**Pull request merged**